AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -171,6 +181,23 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
 - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret
                                           releaseHash:(NSString *)releaseHash
                                           isTesterApp:(BOOL)isTesterApp;
+/**
+ * Configure Distribute options before the service starts.
```

Comment:
```suggestion
 * Configure distribute options before the service starts.
```

Reply:
This is a name of the module so it started with an uppercase.